docs/cliapi.rst

```diff
@@ -6,6 +6,6 @@ CLI API
 Please use ``cernopendata-client --help`` in your terminal shell to learn about
 all commands and their options.
 
-.. click:: cernopendata_client.cli:cernopendata_client
```

Comment:
Now that #78 is merged, shall we try to simply plug back the old ReadTheDocs sphinx doc generation?

Please test locally using `readthedocs/build:latest` build image, because it may need to introduce some more `autodoc_mock_imports` in `docs/conf.py` with respect to both `pycurl` and `xrootd` packages.

Reply:
done 👍